File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    a = num_list[-1]
    b = num_list[-2]
    c = 0
    
    if a > b:
        c = a - b
        num_list.append(c)
        return num_list 
    else:
        c = a * 2
        num_list.append(c)
        return num_list 
    
#list(c)에서 c가 정수형이면 TypeError가 발생합니다. 이는 list 함수가 반복 가능한 객체만을 인자로 받아야 하기 때문입니다. 
# 즉 반복 가능한 객체는 튜플,문자열 등

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.js
    function solution(num_list) {
    var answer = [];
    let last1 = num_list[num_list.length - 1];  // 배열의 마지막 요소를 참조
    let last2 = num_list[num_list.length - 2];  // 배열의 두 번째 마지막 요소를 참조
    
    if (last1 > last2) {
        num_list.push(last1 - last2);  // 마지막 요소가 두 번째 마지막 요소보다 크면 차이를 추가
    } else {
        num_list.push(last1 * 2);  // 그렇지 않으면 마지막 요소의 두 배를 추가
    }  
    
    return num_list;
}
